docs/proposed/psa-driver-interface.md

```diff
@@ -191,6 +191,8 @@ The signature of a driver entry point generally looks like the signature of the
 
 Some entry points are grouped in families that must be implemented as a whole. If a driver supports an entry point family, it must provide all the entry points in the family.
 
+Drivers can also have entry points related to random generation. For transparent drivers, these are [random generation entry points](#random-generation-entry-points). For opaque drivers, these are [entropy collection entry points](#entropy-collection-entry-point).
```

Comment:
Why the split? Neither of these have anything to do with key material, so shouldn't both be in the realm of transparent drivers?

Reply:
Good point. It's closer to opaque drivers because you can have multiple entropy sources, whereas there is a single random generator. But it's only a very minor complication to allow entropy sources in transparent drivers.